code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -273,4 +231,4 @@
 	if(!SS.transfer_soul("FORCE", user)) //Something went wrong
 		qdel(SS)
 		return
-	user.visible_message("<span class='danger'>[user.name]'s soul is captured by \the [src]!</span>", "<span class='userdanger'>You've lost the gamble! Your soul is forfiet!</span>")
\ No newline at end of file
+	user.visible_message("<span class='danger'>[user.name]'s soul is captured by \the [src]!</span>", "<span class='userdanger'>You've lost the gamble! Your soul is forfiet!</span>")

```

Comment:
Nothing to do with me, that's just a line ending change.

Suppose I can fix it as well